website/docs/r/alert_policy.html.markdown

```diff
@@ -26,7 +26,7 @@ The following arguments are supported:
 
   * `name` - (Required) The name of the policy.
   * `incident_preference` - (Optional) The rollup strategy for the policy.  Options include: `PER_POLICY`, `PER_CONDITION`, or `PER_CONDITION_AND_TARGET`.  The default is `PER_POLICY`.
-  * `channel_ids` - (Optional) An array of channel IDs (integers) to assign to the policy. Adding or removing channel IDs from this array will result in a new alert policy resource being created and the old one being destroyed. Also note that channel IDs _cannot_ be imported via `terraform import` (see [Import](#import) for info).
+  * `channel_ids` - (Optional) **DEPRECATED** The `channel_ids` argument is deprecated and will be removed in the next major release of the provider. An array of channel IDs (integers) to assign to the policy. Adding or removing channel IDs from this array will result in a new alert policy resource being created and the old one being destroyed. Also note that channel IDs _cannot_ be imported via `terraform import` (see [Import](#import) for info).
```

Comment:
@sanderblue the documentation is unclear on what to use instead of this deprecated argument. Examples in this resource page are still using deprecated `newrelic_alert_channel` instead of `newrelic_notification_channel`. How do we link policies and channels if we cannot use this deprecated argument?